bitbots_ball_filter/scripts/ball_sim.py

```diff
@@ -49,6 +49,7 @@ def gen_msg(self, x, y):
 
 
 if __name__ == '__main__':
-    SimBall()
+    rclpy.spin(SimBall())
```

Comment:
This should work, but the standard format looks like this:

https://github.com/bit-bots/bitbots_world_model/blob/47e8de43abaab5bdb3f6aa8d85c61078f5e4e494/bitbots_ball_filter/bitbots_ball_filter/ball_filter.py#L258